src/Sfa.Das.Sas.Tools.MetaDataCreationTool/Services/LarsDataService.cs

```diff
@@ -351,7 +356,7 @@ private void AddQualificationsToFrameworks(LarsMetaData metaData)
                                                                       x.ProgType.Equals(framework.ProgType) &&
                                                                       x.PwayCode.Equals(framework.PwayCode)).ToList();
 
-                frameworkAims = frameworkAims.Where(x => x.EffectiveTo >= DateTime.Now || x.EffectiveTo == null).ToList();
+                frameworkAims = frameworkAims.Where(x => x.EffectiveTo >= DateTime.Now || x.EffectiveTo == null || IsSpecialFramework($"{x.FworkCode}-{x.ProgType}-{x.PwayCode}")).ToList();
 
                 var qualifications =
                     (from aim in frameworkAims
```

Comment:
need to add the condition inside method AddDurationAndFundingToFrameworks line 444
`(!fwk.EffectiveTo.HasValue || fwk.EffectiveTo.Value.Date >= DateTime.UtcNow.Date || IsSpecialFramework($"{x.FworkCode}-{x.ProgType}-{x.PwayCode}")));`

Reply:
there is a chance that it will return 2 or more rows then we need to take the latest one.